spec/controllers/maps_controller_spec.rb

```diff
@@ -0,0 +1,49 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe MapsController, type: :controller do
```

Comment:
Can you replace this with request specs?  Controller specs are more or less deprecated at this point in the community.

Reply:
Replaced.